src/cmd/src/cli.rs

```diff
@@ -31,7 +31,6 @@ impl Instance {
     }
 
     pub async fn stop(&self) -> Result<()> {
-        // TODO: handle cli shutdown
```

Comment:
Is it done?

Reply:
> Is it done?

Yep, the repl handled it.
https://github.com/GreptimeTeam/greptimedb/blob/f712f978cfccb7ef0b95486bc49a2155d1a9d502/src/cmd/src/cli/repl.rs#L105-L118